Descension/Assets/Scripts/Actor/Player/PlayerController.cs

```diff
@@ -102,8 +110,15 @@ void Start()
 
             _rb.MovePosition(_rb.position + _rawInputMovement * movementSpeed);
 
-            if (torchQuantity > 0) {
-                torchQuantity -= 1 * Time.deltaTime;
+            if (_torchToggle) {
+                if (torchQuantity > 0) {
+                    torchQuantity -= 1 * Time.deltaTime;
+                    _postProcessing.SettVignetteIntensity(0.5f);
+                } else {
+                    _postProcessing.SettVignetteIntensity(0.9f);
+                }
+            } else {
+                _postProcessing.SettVignetteIntensity(0.9f);

```

Comment:
```suggestion
            // TODO: Refactor to use a constant or variable instead of magic numbers
            if (_torchToggle) {
                if (torchQuantity > 0) {
                    torchQuantity -= 1 * Time.deltaTime;
                    _postProcessing.SettVignetteIntensity(0.5f);
                } else {
                    _postProcessing.SettVignetteIntensity(0.9f);
                }
            } else {
                _postProcessing.SettVignetteIntensity(0.9f);

```